source/tests/pt/test_lmdb_dataloader.py

```
@@ -312,6 +346,83 @@ def test_full_epoch(self, lmdb_dir):
+                batch for dataloader in ds.dataloaders for batch in dataloader
+            ]
+        assert {
+            (float(batch["find_energy"]), float(batch["find_force"]))
+            for batch in stat_batches
+        } == {(1.0, 0.0), (0.0, 1.0)}
+        for batch in batches:
+            flags = (float(batch["find_energy"]), float(batch["find_force"]))
+            observed_flags.add(flags)
+
+            def zero_energy_model(**kwargs):
+                return {"energy": torch.zeros_like(batch["energy"])}
```

Ruff (0.15.21)

[warning] Function definition does not bind loop variable `batch`

(B023)